.github/workflows/ci.yaml

```diff
@@ -138,7 +138,7 @@ jobs:
           export GOPATH=$(go env GOPATH)
           export PATH=${PATH}:$GOPATH/bin
           make verify lint/templates binary test test/integration
-        timeout-minutes: 14
```

Comment:
Why is 14 minutes not enough? Is it the CI runner being slow or something changed on our hand that we need to bump the timeout to a bigger value? In all the case, if this change is required, let's also consider bumping https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/Makefile#L296

Reply:
> Why is 14 minutes not enough? 

Please see the description of the PR.

Reply:
@machi1990 Please see https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1634